src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -603,6 +603,55 @@ LedgerMaster::clearLedger(std::uint32_t seq)
     mCompleteLedgers.erase(seq);
 }
 
+bool
+LedgerMaster::isValidated(ReadView const& ledger)
+{
+    if (app_.config().reporting())
+        return haveLedger(ledger.seq());  // TODO Is this correct ?
```

Comment:
@ximinez @seelabs I am not sure if this is correct, will appreciate feedback. I got the idea from `getValidatedRange` but perhaps a better check would be `return ledger.seq() <= getValidLedgerIndex()`.

FWIW The old `isValidated` in `RPCHelper` would just `return true;` but that did not seem right to me.

Reply:
I switched to `return true;` with a comment explainig the reason.

Reply:
are you still referring to line 610? github shows me the old code, perhaps your new code has not been pushed into github